integration-tests/contracts/contract_vrf_models.go

```diff
@@ -262,12 +262,13 @@ type RequestStatus struct {
 }
 
 type LoadTestRequestStatus struct {
-	Fulfilled             bool
-	RandomWords           []*big.Int
-	requestTimestamp      *big.Int
-	fulfilmentTimestamp   *big.Int
-	requestBlockNumber    *big.Int
-	fulfilmentBlockNumber *big.Int
+	Fulfilled   bool
+	RandomWords []*big.Int
+	// Unused???
```

Comment:
Are these planned to be used @iljapavlovs

Reply:
for now its not used, but potentially I could add these to the Grafana board, but I guess, "as long as we dont use, remove it"